test/test_other.py

```diff
@@ -15209,6 +15209,11 @@ def test_wasm64_no_asan(self):
     err = self.expect_fail([EMCC, test_file('hello_world.c'), '-sMEMORY64', '-fsanitize=address'])
     self.assertContained('error: MEMORY64 does not yet work with ASAN', err)
 
+  def test_mimalloc_no_asan(self):
```

Comment:
Are there other mimalloc tests that this can be grouped with?

Reply:
I grouped this with the other asan test right above, which it is parallel to?